persistence/azure-table/transactions_intro_astp_[5,).partial.md

```diff
@@ -0,0 +1 @@
+By default, the persistence does not attempt to atomically commit saga data and/or business data and uses the saga id as partition key to store sagas. Through the use of the [TableBatchOperation API](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.cosmos.table.tablebatchoperation?view=azure-dotnet), saga data and/or business data can be atomically committed if everything is stored in the same partition within the same table.
```

Comment:
This partial is still referring to `Microsoft.Azure.Cosmos.Table.TableBatchOperation`